app/controllers/api/v1/application_users_controller.rb

```diff
@@ -92,6 +92,24 @@ def index
   end
 
   ###############################################################
+  # find_by_username
+  ###############################################################
+
+  api :GET, '/application_users/username/:username',
+      'Gets a single ApplicationUser with the specified username.'
+  description <<-EOS
+    #{json_schema(Api::V1::UserSearchRepresenter, include: :readable)}
+  EOS
+  def find_by_username
+    application_user = ApplicationUser.includes(:user).joins(:user).where({
+      :user           => { :username => params[:username] },
+      :application_id => current_api_user.application.id
+    }).first!
+    OSU::AccessPolicy.require_action_allowed!(:read, current_api_user, application_user)
+    respond_with [application_user], represent_with: Api::V1::ApplicationUsersRepresenter
```

Comment:
Should be the singular representer `ApplicationUserRepresenter`, then also don't need to wrap the `application_user` in an array on the left there.